src/components/expansion/ExpansionRules.svelte

```diff
@@ -46,8 +54,33 @@
       sortable: true,
     },
     { field: 'authoring_mission_model_id', filter: 'number', headerName: 'Model ID', sortable: true },
-    { field: 'created_at', filter: 'text', headerName: 'Created At', resizable: true, sortable: true },
-    { field: 'updated_at', filter: 'text', headerName: 'Updated At', resizable: true, sortable: true },
+    {
+      field: 'created_at',
+      filter: 'text',
+      headerName: 'Created At',
+      resizable: true,
+      sortable: true,
+      valueGetter: (params: ValueGetterParams<ExpansionRule>) => {
+        if (params.data?.created_at) {
+          return getShortISOForDate(new Date(params.data?.created_at));
+        }
+      },
+    },
+    { field: 'owner', filter: 'text', headerName: 'Owner', resizable: true, sortable: true },
+    {
+      field: 'updated_at',
+      filter: 'text',
+      headerName: 'Updated At',
+      resizable: true,
+      sortable: true,
+      valueGetter: (params: ValueGetterParams<ExpansionRule>) => {
+        if (params.data?.updated_at) {
+          return getShortISOForDate(new Date(params.data?.updated_at));
+        }
+      },
+    },
+    { field: 'updated_by', filter: 'text', headerName: 'Updated By', resizable: true, sortable: true },
+    { field: 'description', filter: 'text', headerName: 'Description', resizable: true, sortable: true },
```

Comment:
@camargo related to another comment you made – should I add `name` to Expansion Rules?

Reply:
Ah nice, yeah I think so!

Reply:
Definitely pushing it on the number of fields here 
<img width="1149" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/5bd6a8e2-1e56-4dc9-bf67-f9cd29e4aa99">

Reply:
To save space I don't think we need to show them all by default. Like the date ones are not as important.

Reply:
Only catch is that we don't have the field picker on this table right now so those dates wouldn't be accessible if hidden?

Reply:
I feel like the `created_at` is the least useful. Could we remove those columns in general? Do users want to see those values?

Reply:
Yeah if updated date is always created date or newer then is there really a need to show it here?

Reply:
> Do users want to see those values?

No, not typically.

Reply:
Ok so nix both dates here? And what about expansion sets, expansion runs, sequences, constraints, and scheduling tables?

Reply:
Yeah I'm fine with hiding the dates from all these tables for now. We need an additional ticket to add the menu to the tables so we can show the hidden columns.

Reply:
Sounds good.

Reply:
https://github.com/NASA-AMMOS/aerie-ui/issues/706